environments/gcp/ingress.tf

```diff
@@ -0,0 +1,77 @@
+module "nginx-controller" {
+  source  = "terraform-iaac/nginx-controller/helm"
+  version = "2.3.0"
+  namespace = "kube-system"
+depends_on = [ module.gke ]
+}
+
+data "http" "manifestfile" {
+  url = "https://github.com/cert-manager/cert-manager/releases/download/v1.13.2/cert-manager.crds.yaml"
+}
+
+data kubectl_file_documents "cert-manager-manifest" {
+  content = data.http.manifestfile.response_body 
+
+}
+
+#Installing cert-manager CRDs from the manifest instead of using Helm is the recommended approach for production environments. 
+#Learn more https://cert-manager.io/docs/installation/helm/#option-1-installing-crds-with-kubectl
+
+resource kubectl_manifest "cert-manager-crds" {
+ for_each = data.kubectl_file_documents.cert-manager-manifest.manifests
+ yaml_body = each.value
+depends_on = [ module.gke ]
+}
+
+resource kubernetes_namespace "cert-manager-ns" {
```

Comment:
Minor:
Could be created directly in the helm release resource via `create_namespace`.